### ISSUE TITLE
Gosimple didn't seems to work even if enabled

### ISSUE THREAD

**Issue by osiridecascioli** [golangci/golangci-lint#165]
I've a file:
-->a.go
```
package test
func t() error{
    if err:= t(); err!=nil{
        return err
    }
    return nil
}
```
then I execute:
>gosimple a.go
**a.go:4:5: 'if err != nil { return err }; return nil' can be simplified to 'return err' (S1013)**

but if I execute:
> golangci-lint run --no-config --disable-all --enable=gosimple a.go 
**Congrats! No issues were found.**

there's something I'm missing?

**Comment by jirfag** [comment]
hi!
we use newer version of `gosimple`, just update it. See [this commit](https://github.com/dominikh/go-tools/commit/57ee65dc7ff868f8fe21064b05f9e744a73bc9ab) for details why this check was disabled
